### PR TITLE
Yamllint HTML output fix

### PR DESCRIPTION
**Pull request by penenadpi**
Applying this commit will:
- make yamllint results appear not only in JSON, but in HTML as well

The following issues will be closed:
- #41

### CODE REVIEW

**Review by RedekG — APPROVED** [review]
Works fine, thanks @penenadpi! Feel free to merge.

**Comment by RedekG** [comment]
@penenadpi  please resolve conflicts.

**Comment by penenadpi** [comment]
Resolved the conflict

File: src/iac_scan_runner/results_summary.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def summarize_outcome(self, check: str, outcome: str, scanned_files: dict, compa
         elif check == "cloc":
             self.outcomes[check]["status"] = "Info"
             return "Info"
-
+
```
Comment (RedekG) [inline src/iac_scan_runner/results_summary.py:184]: Please remove this empty space

**Review by RedekG — DISMISSED** [review]

**Review by RedekG — APPROVED** [review]

**Pull request merged**